### DISCUSSION TITLE
Is commercial usage allowed?

### DISCUSSION THREAD

**Discussion by hardikdava** [open-mmlab/mmyolo#457] [Q&A]
Hello all, MMYolo is a great idea to integrate Yolo family. MMYolo is licensed under GPL-3.0. Are the model trained by MMYolo allowed for commercial usage without disclousing the source code of application in condition that the inference script is custom developed but only the model is trained using MMYolo? 
Thank you in advance.

**Comment by RangeKing** [comment]
We will discuss this question and will get back to you in a timely manner.

**Comment by RangeKing** [comment]
@hardikdava, we suggest to use RTMDet which is allowed for commercial usage without disclousing the source code. If you want to use YOLOv5 and YOLOv8, it might be better to request an Enterprise License at Ultralytics Licensing.

**Reply by asutic** [reply]
@RangeKing, @hhaAndroid could you elaborate on the use of RTMDet? Is it okay to use RTMDet from this repository on a custom dataset for commercial purposes, or do users have to pull RTMDet from mmdetection repository?

**Reply by RangeKing** [reply]
@asutic it's okay to use RTMDet both in MMYOLO and MMDet.

**Reply by asutic** [reply]
Thank you for clarifying. Appreciated!

**Reply by MickaMickaMicka** [reply]
> @hardikdava, we suggest to use RTMDet which is allowed for commercial usage without disclousing the source code. If you want to use YOLOv5 and YOLOv8, it might be better to request an Enterprise License at Ultralytics Licensing.

Does this mean you don't consider the training result in general being a derivative work of MMYOLO (like an edited image is not a deeivative work of the image editor software)? But the input of the training (like a yolo v5 model architecture description or the used image data) matter (because the trained model is derived from those)?
